consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/impl/BreathingSpaceServiceTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.moj.sdt.cmc.consumers.client.impl;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.cmc.consumers.api.CMCApi;
+import uk.gov.moj.sdt.cmc.consumers.model.breathingspace.BreathingSpaceRequest;
+import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class BreathingSpaceServiceTest extends AbstractSdtUnitTestBase {
+
+    @Mock
+    private CMCApi cmcApi;
+
+    private BreathingSpaceService breathingSpaceService;
+
+    @BeforeEach
+    @Override
+    public void setUp() {
+        breathingSpaceService = new BreathingSpaceService(cmcApi);
+    }
+
+    @Test
+    void getClient() {
+        BreathingSpaceRequest breathingSpaceRequest = mock(BreathingSpaceRequest.class);
+        breathingSpaceService.breathingSpace(breathingSpaceRequest);
+        verify(cmcApi).breathingSpace(anyString(), anyString(), any(BreathingSpaceRequest.class));
```

Comment:
removed this method

Reply:
For the remaining breathing space parameter you could use the breathingSpaceRequest object you've set up rather than any(BreathingSpaceRequest.class), i.e. ```verify(cmcApi).breathingSpace(breathingSpaceRequest);```.  It is the object that cmcApi.breathingSpace() is expected to be called with, so it makes sense to use it.

Reply:
done